### ISSUE TITLE
Allow for filtering by minimum particle size analyzed, could even filter the particle values themselves to that size range. (Medium priority)

### ISSUE THREAD

**Issue by wincowgerDEV** [Moore-Institute-4-Plastic-Pollution-Res/Microplastic_Data_Portal#117]

**Comment by wincowgerDEV** [comment]
We think this should be its own issue with a longer horizon. We want to have more real data in the tool before we start experimenting with how best to accurately display it based on particle size bias. Probably in a year or two when this tool has enough data we will also have a better idea of how to transform it in meaningful ways.

**Comment by wincowgerDEV** [comment]
Also adding this. "Create a gui for data download to help people filter. (low priority)"
